perf/tabulate_perf_summary.jl

```diff
@@ -27,19 +27,20 @@ all metrics can be found in `get_summary`.
 
 ca_dir = joinpath(dirname(@__DIR__))
 
-function get_job_ids(buildkite_yaml; filter_name = "--perf_summary true")
-    buildkite_commands = commands_from_yaml(buildkite_yaml; filter_name)
+function get_job_ids(buildkite_path; filter = nothing)
+    buildkite_commands =
+        configs_per_job_id(buildkite_path; filter, include_perf = true)
     @assert length(buildkite_commands) > 0 # sanity check
-    job_ids = map(buildkite_commands) do bkcs
-        strip(first(split(last(split(bkcs, "--job_id ")), " ")), '\"')
-    end
+    job_ids = [
+        dict["job_id"] for dict in buildkite_commands if haskey(dict, "job_id")
+    ]
     return job_ids
 end
 
 function combine_PRs_performance_benchmarks(path)
     job_ids = get_job_ids(
-        joinpath(ca_dir, ".buildkite", "pipeline.yml");
-        filter_name = "--perf_summary true",
+        joinpath(ca_dir, ".buildkite");
+        filter = ("perf_summary" => true),
```

Comment:
I prefer the former name (`filter_name`), or something similarly descriptive to avoid overwriting the existing Base function (`Base.filter`)